src/geo/transform.js

```diff
@@ -1269,6 +1269,7 @@ class Transform {
     }
 
     _getBounds(min: number, max: number) {
+        assert(min < Number.MAX_VALUE);
```

Comment:
nit: Should we leave this assert or is it somewhat redundant to [this one](https://github.com/mapbox/mapbox-gl-js/pull/11339/files#diff-a2b3330768267b8c7996b8bf292fcd864e8b768f538436813b0d7e19f7202176R1314)? It seems to cover the same, when we basically fail to retrieve the `minmax` from the DEMs.

Reply:
Good point Karim! My intention was to protect against potential changes or additions to this code in the future, since the new unit test would rely on these assertions to catch a regression. But considering that this is a private function, I think we can safely consider this redundant.